winml/lib/Api/LearningModelSession.cpp

```diff
@@ -89,9 +88,6 @@ LearningModelSession::GetOptimizedModel(bool should_close_model) {
 }
 
 void LearningModelSession::Initialize() {
-  // Begin recording session creation telemetry
-  _winmlt::TelemetryEvent session_creation_event(
-    _winmlt::EventCategory::kSessionCreation);
```

Comment:
This event doesn't send up telemetry. It merely logs a runtime ETW event. This is because there is no telemetry keyword in the TraceLoggingWrite call. An example of telemetry keyword is: MICROSOFT_KEYWORD_MEASURES, MICROSOFT_KEYWORD_TELEMETRY, or MICROSOFT_KEYWORD_CRITICAL_DATA.

I think it would be useful to still have a ETW tracelogging write event that indicates that a session creation in WinML layer for debugging purposes.